src/lib/requests.ts

```diff
@@ -43,19 +43,16 @@ import {
 
 // Config file to instantiate ThirdPartyRequest object
 const configRequest: BaseRequestConfigType = {
-  dfspId: Config.PARTICIPANT_ID as string,
+  dfspId: Config.PARTICIPANT_ID,
   logger: Logger,
-  // TODO: Decide on below later - Handled in future ticket #361
-  // Also decide on need for jwsSigningKey
-  jwsSign: false,
-  tls: {
-    outbound: {
-      mutualTLS: {
-        enabled: false
-      }
-    }
-  }
-
+  peerEndpoint: Config.SHARED.PEER_ENDPOINT,
+  alsEndpoint: Config.SHARED.ALS_ENDPOINT,
+  quotesEndpoint: Config.SHARED.QUOTES_ENDPOINT,
+  transfersEndpoint: Config.SHARED.TRANSFERS_ENDPOINT,
+  bulkTransfersEndpoint: Config.SHARED.BULK_TRANSFERS_ENDPOINT,
```

Comment:
these params are used here: https://github.com/mojaloop/sdk-standard-components/blob/master/src/lib/requests/baseRequests.js#L125-L135

so you need to add all `thirdpartyRequestsEndpoint` 
take a look also here: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/src/server/plugins/state.ts#L83-L111